chef_master/source/knife_setup.rst

```diff
@@ -0,0 +1,131 @@
+=====================================================
+Setting up Knife
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/knife_using.rst>`__
+
+The knife command line tool must be configured to communicate with the Chef Server as well as any other infrastructure within your organization. This is done initially during the workstation setup, but subsequent modifications can be made using the knife.rb configuration file.
+
+Knife.rb Configuration File
+=====================================================
+
+Knife is configured using a knife.rb configuration, which contains configuration for both the knife command line tool as well as any installed knife plugins. See :doc:`knife.rb </config_rb_knife>` for a complete list of configuration options in the knife.rb file.
```

Comment:
This is linking to the `config_knife_rb` file, but that file is redirected to `knife using` ..is the intention to get rid of `config_knife_rb` altogether?

Reply:
Let me check on that.

Reply:
I just built locally and this links to the doc with all the common config items. That should be good